mathml/relations/html5-tree/math-global-event-handlers.tentative.html

```diff
@@ -42,6 +42,9 @@
           );
         }, `${name}: must be on the appropriate locations for GlobalEventHandlers`);
 
+        if (!location.hasOwnProperty(name))
```

Comment:
I think this is effectively skipping tests (and hence reducing the test count)?
Would it be possible instead to modify the async_test to add a t.step() call that asserts MathMLElement.prototype.hasOwnProperty(name) ? IIUC, failure would just stop the test without waiting for the t.step_func_done().

BTW, I think we want the same for clipboard-event-handlers.tentative.html

Reply:
Funnily enough this exposes another test and increases test count:
-Found 244 tests; 238 PASS, 6 FAIL, 0 TIMEOUT, 0 NOTRUN.
+Found 297 tests; 295 PASS, 2 FAIL, 0 TIMEOUT, 0 NOTRUN.

I don't see the point in running subtests we have already deduced will fail.

Reply:
Mmh, it seems you are comparing against your initial commit. This is what I get comparing with ToT:

Current ToT
Found 305 tests
295 Pass
8 Fail
2 Timeout

With the patch below
Found 305 tests
295 Pass
10 Fail

diff --git a/mathml/relations/html5-tree/math-global-event-handlers.tentative.html b/mathml/relations/html5-tree/math-global-event-handlers.tentative.html
index e96feeaf09..859feff422 100644
--- a/mathml/relations/html5-tree/math-global-event-handlers.tentative.html
+++ b/mathml/relations/html5-tree/math-global-event-handlers.tentative.html
@@ -127,2 +127,5 @@
         async_test(t => {
+          t.step(function() {
+              assert_true(MathMLElement.prototype.hasOwnProperty(name));
+          });
           const element = document.createElementNS(

I think we should not reduce the number of tests (305 tests here) and moreover we should keep the same number of tests for all browsers. Skipping tests according to whether the property is supported by the browser will make the percentage of pass incomparable in the implementation report. I we cannot avoid this issue when the browser crashes but here we have a simple solution.